Sources_v3/Models/ChannelType.swift

```diff
@@ -47,11 +43,6 @@ public enum ChannelType: Codable, Hashable {
     ///
     /// - Parameter rawValue: a string raw value of a channel type.
     init(rawValue: String) {
-        if rawValue.isEmpty || rawValue.contains(" ") || rawValue == "unknown" {
-            self = .unknown
-            return
-        }
-        
```

Comment:
?

Reply:
I don't want to make `ChannelType` optional on `ChannelId` or throwable `init` cause we don't have any fallback for this check.
I made `ChannelId` init more safe instead. Check 13ffaab7.